build/charts/theia/values.yaml

```diff
@@ -131,6 +131,8 @@ clickhouse:
     # Specification to install a default ZooKeeper cluster. This is ignored
     # if zookeeperHosts are provided. 
     installZookeeper:
+      # -- Determine whether to enable persistent volume for zookeeper
+      createPersistentVolume: false
```

Comment:
As mentioned above, if we move the ZooKeeper storage section here, I think it's not necessary to have this boolean createPersistentVolume. Instead we can read from storage.createPersistentVolume.type to determine whether PV creation is required, as what we do for ch and grafana

Reply:
I forgot to delete it. Will remove it later.